### DISCUSSION TITLE
support for multiple windows in columns?

### DISCUSSION THREAD

**Discussion by cori** [rxhanson/RectanglePro-Community#333] [Q&A] [answered]
I'd love to be able to use a multiple windows function to spread my windows in columns rather than (or in addition to) tiles - so take the windows on a monitor and best-guess them into first third, center third, last third, or similarly with quarters (i have a pretty ginormous monitor, and a few full-height, partial-width windows works pretty well). I don't see anything built-in for that, and I assume there isn't a workaround, but i'd be happy to see it added to future features (or to find out i can hack my way into it).

**Comment by rxhanson** [comment]
I don't think I know precisely what you're looking for, but typically the app layouts functionality is what most users use for lining up multiple windows. The easiest way to do this is to position all your windows in the places you want and select save layout from the menu bar menu. You can change each individual window layout to use screen percentages (values less than 1) and docking areas. You could change the selected app on all of them to "Global" if you don't care where your windows go, just that they are arranged in columns. 

I could see adding a multi-window layout for columns, it just seems that most people are concerned with which app windows go where, rather than assigning them based on current position or z-index.

**Comment by cori** [comment]
aye it's a bit hard to describe, and I always feel like I'm a bit of an edge case with most things, so this may be one of those situations, but i'll try to be a bit clearer, this time with visuals :-D

First I am a heavy user of app layouts - i have maybe a dozen, many with shortcuts programmed into my stream deck. I also hav one ginormous monitor - a 48" 16:9, so that may have a lot to do with my ask here.

It seems like you have an idea of what I'm looking for, but for clarity I'm looking for an addition to the multiple windows menu item. alongside

<img width="424" alt="image" src="https://github.com/rxhanson/RectanglePro-Community/assets/46317/78d5c2f1-42c0-4c57-859d-7cd4813e732c">

which results in e.g. 

<img width="2048" alt="image" src="https://github.com/rxhanson/RectanglePro-Community/assets/46317/06b2401d-5323-4e21-9392-0cd2503195bc">

I'd like to see one that's three columns and results in e.g.

<img width="2048" alt="image" src="https://github.com/rxhanson/RectanglePro-Community/assets/46317/33ae1c68-1efb-482e-943a-f4c48f61bd4e">

that shot might be a little misleading since it's from my laptop screen, but on my main monitor that results in three very usable windows. If I'm, say, in training like I am this week, I don't necessarily want to create an app layout, but if I could get my three important windows on the same desktop and then execute the column layout shortcut I'd be golden for all kinds of ad hoc configuration of semi-arbitrary apps.

**Reply by rxhanson** [reply, marked as answer]
Thanks for the additional info. Did you try making an app layout and just setting all of the apps to "Global"? I realize that's not the same as having it in there as a preset, but I think it should get what you're looking for? You could add it to the menu from the Menu tab in the prefs window. 

I don't mind adding this in, but it'll take me some time to get to it.

**Reply by cori** [reply]
no rush at all - will be nice to have but I can totes work around it for now. thanks!

**Reply by jmalczak** [reply]
Hi @rxhanson. I have tried solution you have proposed. I set all of the apps to Global. One app was 1/3 docking position top left, second 1/3 docking position custom 1/3 and third one 1/3 and docking position top right. When I have 3 different apps opened and run configured app layout. All of the apps ends up in first 1/3.
